### PR TITLE
Antavo Loyalty Cloud - Partner Docs

### PR DESCRIPTION
**Pull request by andrasAntavo**
# Pull Request/Issue Resolution

CC @josh-mccrowell-braze @bre-fitzgerald

Please reach out to Tibor Tasi ([tibor.tasi@antavo.com](mailto:tibor.tasi@antavo.com)) and Gabor Vigh ([gabor.vigh@antavo.com](mailto:gabor.vigh@antavo.com)) for details.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [X] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [X] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [X] Check that all links work.
- [X] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [X] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [X] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by andrasAntavo** [comment]
@josh-mccrowell-braze @bre-fitzgerald @zzhaobraze as discussed, here is the new PR for the Antavo Loyalty Cloud partner docs with all the previous changes (including the ones made by you) from the previous PR. It seems that the CLA issue has been resolved by this.

**Comment by bre-fitzgerald** [comment]
Thanks @andrasAntavo!

**Review by bre-fitzgerald — APPROVED** [review]

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/antavo.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+4. To add additional field pairings, repeat steps 1–3.
+5. To remove a field from the list of synchronized data, click <i class="fa-solid fa-rectangle-xmark" title="Delete"></i> at the end of the row.
+6. Click **Save**.
+
+When any value of the configured fields changes in Antavo, not only the synchronization of that single value is triggered, but every field added to the field mapping is included in the request.
+
+![Sync Fields page in Antavo.][2]
+
+{% alert important %}
+To ensure you minimize data point consumption, we recommend only mapping the fields that will be actioned on within Braze.
```
Comment (rachel-feinberg) [inline _docs/_partners/message_orchestration/channel_extensions/loyalty/antavo.md:67]:
```suggestion
To minimize data point consumption, we recommend only mapping the fields that will be actioned on within Braze.
```
Avoid using "ensure"

**Review by rachel-feinberg — APPROVED** [review]
Left a comment about removing "ensure", but looks ready to go

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/antavo.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_partners/message_orchestration/channel_extensions/loyalty/antavo.md]: Thanks for catching this!

**Review by josh-mccrowell-braze — APPROVED** [review]
Updated submitted documentation to align with other Braze documentation. Your documentation will be added to the site once we get the go-ahead from our partnership team. Please feel free to reach out if you have any issues. 

Thanks, The Braze Documentation Team

**Pull request merged**